scripts/init/opx-create-interface.service

```diff
@@ -1,16 +1,16 @@
 [Unit]
 Description=This service is to create all interface during system initiation
 ConditionPathExists=!/etc/opx/nas_if_nocreate
-After=opx-cps.service opx-nas.service opx-front-panel-ports.service
-Requires=opx-cps.service opx-nas.service opx-front-panel-ports.service
+After=opx-cps.service opx-nas.service
+Requires=opx-cps.service opx-nas.service
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
 EnvironmentFile=/etc/opx/opx-environment
 ExecStart=/usr/bin/python -u /usr/bin/base_nas_create_interface.py
-ExecStartPost=/usr/bin/python -u /usr/bin/base_nas_fanout_init.py && "/bin/sh -c /usr/bin/network_restart.sh"
-TimeoutStartSec=180
+ExecStartPost=/bin/sh -c "/bin/sh -c /usr/bin/base_nas_fanout_init.sh && /usr/bin/network_restart.sh"
```

Comment:
Revert this change. Refer https://github.com/open-switch/opx-nas-interface/commit/78168e4e69fc99ca635dea9962d453a886d1e94c

Reply:
Please address this.